codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/EnumGenerator.kt

```diff
@@ -108,6 +111,10 @@ open class EnumGenerator(
             // pub enum Blah { V1, V2, .. }
             renderEnum()
             writer.insertTrailingNewline()
+            if (target == CodegenTarget.CLIENT) {
+                renderUnknownVariantValue()
+            }
```

Comment:
we should probably add codegenContext.clientOnly { /* ... lambda ... */ } as a helper

Reply:
Thanks for the suggestion! Addressed in 83a15e0.